hw_vm_tools/tasks/installation.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: remove unused tools
+  package:
+    name: '{{ item }}'
+    state: absent
+  with_items: '{{ hw_vm_tools_packages_remove }}'
+  when: hw_vm_tools_packages_remove is defined
+
+- name: remove unused tools on virtual machines
+  package:
+    name: '{{ item }}'
+    state: absent
+  with_items: '{{ hw_vm_tools_vm_packages_remove }}'
+  when: hw_vm_tools_vm_packages_remove is defined
```

Comment:
Maybe add a check for the virtualization_role here as well?

Reply:
Makes sense.